cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1329,6 +1330,26 @@ def make_resource(self, data, **kwargs):
         return Dns(**data)
 
 
+class DatabaseSchema(BaseSchema):
+    """Represent the schema of the Slurm Database."""
+
+    uri = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    user_name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    password_secret_arn = fields.Str(
+        required=True,
+        validate=validate.Regexp(r"^arn:.*:secret"),
+        metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP},
+    )
+    # We don't want to take ownership of the whole structure of the StorageParameters config
+    # parameter in slurmdbd.conf, so we define a generic dictionary rather than a subschema.
+    slurmdbd_storage_parameters = fields.Dict(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
```

Comment:
Leaving a note so that we can discuss whether we want to leave this as is, extend to allow any slurmdbd param to ba passed or drop for now and expect users to modify slurmdbd.conf in this first iteration.

Reply:
Yes, we should discuss this.

Reply:
This is going to be removed from the beta. Resolving this conversation.